index.js

```diff
@@ -136,49 +178,75 @@ function nowServing(line) {
 }
 
 const line = []
+const extraLine = []
 
 // ***** Scope & Closures - Question 1 *****
 
 
 // *** Uncomment the lines below to test
-// console.log("%cScope & Closures - Question 1", "color: red")
+console.log("%cScope & Closures - Question 1", "color: red")
+
+function takeATicketNumber(line){
+  let nextNum = extraLine.length+1
+  line.push(nextNum)
+  extraLine.push(nextNum)
+  return `Welcome, You are ticket number ${nextNum}`
+}
```

Comment:
You are using `extraLine` mostly for its length property, which you increase by 1 in order to get a value for `nextNum`. Then, you are pushing the `nextNum` into the `extraLine` array solely to increase the length of the array.  While this may work, this can be simplified. What is the purpose of storing nextNum in this array (especially if we are never actually using these stored values from the `extraLine` array)? Think about replacing `extraLine` with a variable that holds a number data type instead.